test/api/v3/unit/middlewares/ensureDevelpmentMode.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable global-require */
+import {
+  generateRes,
+  generateReq,
+  generateNext,
+} from '../../../../helpers/api-unit.helper';
+import ensureDevelpmentMode from '../../../../../website/src/middlewares/api-v3/ensureDevelpmentMode';
+import { NotFound } from '../../../../../website/src/libs/api-v3/errors';
+import nconf from 'nconf';
+
+describe('developmentMode middleware', () => {
+  let res, req, next;
+
+  beforeEach(() => {
+    res = generateRes();
+    req = generateReq();
+    next = generateNext();
+  });
+
+  it('returns not found when in production mode', () => {
+    sandbox.stub(nconf, 'get').withArgs('IS_PROD').returns(true);
+
+    ensureDevelpmentMode(req, res, next);
+
+    expect(next).to.be.calledWith(new NotFound());
+  });
+
+  it('passes when not in production', () => {
+    sandbox.stub(nconf, 'get').withArgs('IS_PROD').returns(false);
+
+    ensureDevelpmentMode(req, res, next);
+
+    expect(next).to.be.calledOnce;
+    expect(next.args[0]).to.be.empty;
```

Comment:
I added this expectation, @crookedneighbor . I'm unsure if it is the best way, but it does confirm that NotFound was not called.